src/Propulsion/Streams.fs

```diff
@@ -977,4 +984,4 @@ module Sync =
                     (   dispatcher, maxBatches=maxBatches, maxCycles=defaultArg maxCycles 128, idleDelay=defaultArg idleDelay (TimeSpan.FromMilliseconds 0.5))
```

Comment:
will `TimeSpan.FromMilliseconds 0.5` default to zero `sleepIntervalMs` in `StreamSchedulingEngine`?

Reply:
Yes, it will, see #82 for the fix